Cleanup/editing for admin_guide/quota and fix links

adellape opened the pull request:
General clean-up and editing to this topic, but also follows up on https://github.com/openshift/openshift-docs/pull/1919 (which moved the topic from the Dev Guide to the Admin Guide) to fix a broken link or two introduced by the move.

Also:

- Hides references to configmap for the OSE 3.1 builds for now. Will reintroduce for OSE 3.2 in a separate PR.
- Hide links to the Dev Guide in the Atomic Registry build, since AR doesn't have a Dev Guide.

cc @derekwaynecarr @aweiteka 

Pretty builds:

Openshift Origin - http://file.rdu.redhat.com/~adellape/042916/openshift-origin/fix-admin-quota/admin_guide/quota.html

Atomic Registry - http://file.rdu.redhat.com/~adellape/042916/atomic-registry/fix-admin-quota/admin_guide/quota.html

derekwaynecarr commented on admin_guide/quota.adoc:
```diff
@@ -6,140 +6,169 @@
 
-|`configmaps`
-|The total number of config maps that can exist in the project.
+ifdef::openshift-origin,atomic-registry[]
```
I am not sure when/how we remember to change this when next version drops, but I assume that is something you guys track?

derekwaynecarr replied:
right now, requests.* and limits.* are only in openshift-origin 1.2

adellape replied:
Yeah, we use milestones to track getting newer stuff landed into downstream branches at the right times, but in this PR's case, it's a follow-up to https://github.com/openshift/openshift-docs/pull/1919 where we moved the topic wholesale from Dev Guide -> Admin Guide, which I would like to get fixed in the 3.1 docs as well. So as a result I'm just ifdef'ing out the future stuff for now. I'll track getting this 3.2/1.2 stuff into OSE at the right time separately.

adellape replied:
@derekwaynecarr Ah, now I recall your https://github.com/openshift/openshift-docs/pull/1747 from March. Now I see the 1.2/3.2 changes are more than I thought.

Maybe for downstream actually I won't cherry-pick these commits (or https://github.com/openshift/openshift-docs/pull/1919). I would still like to move the topic in the 3.1 docs, so I might just go do that explicitly on the enterprise-3.1 branch.

derekwaynecarr commented on admin_guide/quota.adoc:
```diff
@@ -6,140 +6,169 @@
 |===
 
+[[quota-scopes]]
 == Quota Scopes
```
quota scopes is new to kube 1.2 and origin 1.2

adellape commented:
@derekwaynecarr OK, thanks for looking. Updated this to remove some of those ifdef's, and I've changed the milestone here (and in Aaron's https://github.com/openshift/openshift-docs/pull/1919) to OSE 3.2, so that it lands at the same time as your https://github.com/openshift/openshift-docs/pull/1747.
